File: libclang/src/main/java/klang/jvm/binding/CXIdxLoc.java

package klang.jvm.binding;

import com.sun.jna.Pointer;
import com.sun.jna.Structure;

@SuppressWarnings("unused")
@Structure.FieldOrder({"ptr_data", "int_data"})
public class CXIdxLoc extends Structure {
    public Pointer[] ptr_data = new Pointer[2];
    public int int_data;

    public static class ByValue extends CXIdxLoc implements Structure.ByValue {}
}

File: libclang/src/main/java/klang/jvm/binding/CXIdxLoc.kt
package klang.jvm.binding

import com.sun.jna.Pointer
import com.sun.jna.Structure

@Suppress("unused")
@Structure.FieldOrder("ptr_data", "int_data")
open class CXIdxLoc : Structure() {
	@JvmField
    var ptr_data = arrayOfNulls<Pointer>(2)
	@JvmField
    var int_data = 0

    class ByValue : CXIdxLoc(), Structure.ByValue
}
